docs/SETUP_SIK.md

```diff
@@ -0,0 +1,23 @@
+# mLRS Documentation: Setup as SiK Telemetry Replacement #
+
+([back to main page](../README.md))
+
+This page describes how to set up an mLRS system to work as a bi-directional serial MAVLink telemetry link, independent on any radio, where mLRS' rc data capability is not desired and mLRS should only transmits and receives serial MAVLink data. This setup finds application in situations where traditionally, e.g., SiK telemetry uints would be used.
+
+<img src="images/mLRS-docu-setup-sik-telemetry-01.jpg" width="800px">
+
+## Setup
+
+The configuration for this mode of operation just follows that for any other configuration, except that one simply ignores the rc data input on the mLRS Tx module and the rc data output on the mLRS receiver.
```

Comment:
How about 'The configuration for this mode is similar to others, except that one can ignore the rc settings on both the Tx and Rx.'

Reply:
done